weblate/trans/views/edit.py

```diff
@@ -141,7 +142,12 @@ def search(translation, request):
     name = form.get_name() if form_valid else ""
 
     # Grab unit IDs
-    unit_ids = list(allunits.order().values_list("id", flat=True))
+    sort_by_list = (
+        request.GET.get("sort_by").split(",")
+        if request.GET.get("sort_by")
+        else ["-priority", "position"]
```

Comment:
I'd rather hard-code available choices here. This way user can do any ordering which might result in huge load on the database and DOS.

Reply:
...maybe also moving such helper method to UnitQuerySet and reuse it in weblate/trans/views/search.py

Reply:
Yeah. Makes sense. I will put it into a helper function with a hardcoded list of allowed sortings.